ql/src/java/org/apache/hadoop/hive/ql/exec/FileSinkOperator.java

```diff
@@ -736,6 +740,21 @@ protected void initializeOp(Configuration hconf) throws HiveException {
     }
   }
 
+  private boolean skipPartitionCheck() {
```

Comment:
Maybe a comment would be good here

Reply:
Done

Reply:
Decided to also rename the method to `readPartitionValues()`